docs/sources/mimir/operators-guide/architecture/components/alertmanager.md

```diff
@@ -97,9 +97,10 @@ When using a reverse proxy, use the following settings when you configure the HT
 
 The Mimir Alertmanager adds some custom template functions to the default ones of the Prometheus Alertmanager.
 
-| Function   | Params | Description                                    |
-| ---------- | ------ | ---------------------------------------------- |
-| `tenantID` | -      | Returns ID of the tenant the alert belongs to. |
+| Function            | Params                                        | Description                                                                                                                                                 |
+| ------------------- | --------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `tenantID`          | -                                             | Returns ID of the tenant the alert belongs to.                                                                                                              |
+| `grafanaExploreURL` | `grafana_URL`,`datasource`,`from`,`to`,`expr` | Returns link to Grafana explore based on the input parameters. Example: `{{ grafanaExploreURL "https://foo.bar" "xyz" "now-12h" "now" "up{foo=\"bar\"}" }}` |
```

Comment:
[nit] Please explain it's a range query.

Reply:
right, added in https://github.com/grafana/mimir/pull/3849/commits/e7abe5cb4c9c324dba89ff53ad3e7f636143cf31